rclcpp/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ if(AMENT_ENABLE_TESTING)
 
   ament_add_gtest(test_mapped_ring_buffer test/test_mapped_ring_buffer.cpp)
   ament_add_gtest(test_intra_process_manager test/test_intra_process_manager.cpp)
+  ament_add_gtest(test_function_traits test/test_function_traits.cpp)
```

Comment:
Please move this new test below or above the existing tests. Currently it is in the middle of the `test_intra_process_manager` test.